linux_os/guide/services/ssh/ssh_server/sshd_disable_root_password_login/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Disable SSH Root Login with a password'
+
+description: |-
+    The root user should not be allowed to login to a
+    system directly over a network with a password.
+    To disable root login via SSH, add or correct the following line
+    in <tt>/etc/ssh/sshd_config</tt>:
+    <pre>PermitRootLogin prohibit-password</pre>
+
+rationale: |-
+    Even though the communications channel may be encrypted, an additional layer of
+    security is gained by preventing use of a password.
+    This also helps to minimize direct attack attempts on root's password.
+
+severity: medium
+
+{{{ complete_ocil_entry_sshd_option(default="prohibit-password", option="PermitRootLogin", value="prohibit-password") }}}
```

Comment:
@jcpunk please expand and do not use `complete_ocil_entry_sshd_option`

Reply:
I'm not sure what this should be expanded to....

I copied the stanzas from `sshd_disable_root_login`

Reply:
```
ocil_clause: 'it is commented out or not configured properly'

ocil: {{{ ocil_sshd_option(default="prohibit-password", option="PermitRootLogin", value="prohibit-password") }}}

Reply:
Just for curiosity, what is the reason not to use the `complete_ocil_entry_sshd_option` macro?

Reply:
It's over-templating, inconsistent, and you shouldn't hide yaml keys behind the templates as it causes problems for non-python tools to process. Which is why other uses of the complete macro have been slowly removed.

Reply:
Will fix shorlty.